signingscript/sign.py

```diff
@@ -240,10 +240,12 @@ def build_signtool_cmd(context, from_, fmt, to=None, servers=None):
         files = await _extract_zipfile(context, orig_path, tmp_dir=tmp_dir)
     else:
         files = [orig_path]
+    files_to_sign = [file for file in files if _should_sign_windows(file)]
+    if not files_to_sign:
+        raise SigningScriptError("Did not find any files to sign, all files: {}".format(files))
```

Comment:
See [this comment](https://github.com/mozilla-releng/signingscript/pull/84#issuecomment-432483543).

Other than confusion about why we're not doing anything and going green, do you see another reason to error out when we have nothing to sign? Do you think it's valid to be able to re-sign an already-signed zipfile and have it noop?

Reply:
Looks like I'm mistaken; if we check for the signature it's in signtool or elsewhere.